05-descriptive-analysis.Rmd

```diff
@@ -526,7 +528,7 @@ Note above that functions `survey_total()`, `survey_mean()`, and `survey_prop()`
 
 Calculating proportions with design effects is important when analyzing survey data that has been collected using a complex sampling design. Since a complex sampling design means that the probability of selecting each individual in the sample is different, this can lead to the observations in the sample being correlated with each other.
 
-Design effects account for this correlation and adjust the standard errors of estimates, which can affect the significance of test statistics and the accuracy of confidence intervals. Calculating proportions with design effects is important because it helps to produce accurate estimates of population proportions that account for the complex nature of the survey data.
+Design effects account for this correlation and adjust the standard errors of estimates, which can affect the significance of test statistics and the accuracy of confidence intervals. Calculating proportions with design effects is important because it helps to produce accurate estimates of population proportions that account for the complex nature of the survey data. 
```

Comment:
I think we need to provide more information around deff.  What do they tell us?  How can we use that in analysis and reporting?  I remember having these same questions when I was learning about them in class so I'm not much help here to flush this out 😄